deployment/mesh-infra/storage/sc/directpv.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: directpv-min-io
+provisioner: kubernetes.io/no-provisioner
+volumeBindingMode: WaitForFirstConsumer
+
+# NOTE
+# ----
+# 1. Provisioner. Weird, but true. The above def is a copy-paste from
+# - https://github.com/minio/operator
```

Comment:
...and it looks like the operator's docs are wrong. In fact, the DirectPV install command generates its own `directpv-min-io` storage class with a proper provisioner. Zap this file.

Reply:
done